Maddy-maddigordon757/Homework-1/fizzbuzz/app.js

```diff
@@ -0,0 +1,38 @@
+
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+for (var i=1; i<100; i++)
+{
+    console.log(i);
+}
+
+
+// ##### Step 2:
+// Add an `if/else` statement that logs the string `"fizz"` if the value being iterated over is divisible by `3`; otherwise, log out the value.
+
+for (var i=1; i<100; i++)
+{
+    console.log(i);
+    if {i % 3 = 0;
```

Comment:
This isn't quite the correct syntax for a conditional statement. Remember that the condition goes in parens, and then the code to execute if that condition is true goes in a block ({ }).